<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Arithmetic Expression Calculator</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QLineEdit" name="leftOperandLineEdit"/>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
              <widget class="QComboBox" name="operationComboBox">
                <item>
                  <property name="text">
                    <string>+</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>-</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>*</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>/</string>
                  </property>
                </item>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="rightOperandLineEdit"/>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QPushButton" name="calculateButton">
            <property name="text">
              <string>Calculate</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QLabel" name="resultLabel">
            <property name="text">
              <string>Result: </string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
